cpp/cmake/modules/ConfigureCUDA.cmake

```diff
@@ -64,6 +64,6 @@ endif()
 # Debug options
 if(CMAKE_BUILD_TYPE MATCHES Debug)
   message(VERBOSE "cuVS: Building with debugging flags")
-  list(APPEND CUVS_CUDA_FLAGS -G -Xcompiler=-rdynamic)
+  list(APPEND CUVS_CUDA_FLAGS -G -Xcompiler=-rdynamic --maxrregcount=64)
```

Comment:
Isnt this the opposite of what we want? The warning says that 64 is too big:
`ptxas warning : Too big maxrregcount value specified 64, will be ignored`

Reply:
This is not to solve the current error on CI, this is a workaround for building in debug

Reply:
Adding that flag makes the debug build succeed so yes it is solving the problem.
This flag might make the kernels slower by reducing the number of register per function but it shouldn't matter that much because the purpose of a debug build is not to be optimal speed-wise.